libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.m

```diff
@@ -69,6 +69,10 @@ + (instancetype)sharedInstanceWithComponent:(NSString *)componentName {
     }
 }
 
++ (instancetype)sharedInstance {
+    return [self sharedInstanceWithComponent:kDefaultComponentName];
```

Comment:
NB: `sharedDefaultInstance` could just go away with this implementation—it'd simply be the default implementation of `sharedInstance`.